doc/RELEASE_NOTES.md

```diff
@@ -154,44 +146,46 @@ Along with this change, 3 new named tool sets have also be added:
  * `heavy`: ${medium}, `perf`, `pidstat`, `proc-interrupts`, `proc-vmstat`, `turbostat`
 
+Users are not required to ues the pre-defined tool sets.  A user should be able to register whatever tools they like, or create their own tool sets by following the pattern of the provided named tool sets.
+
 
 ## Removal of Gratuitous Manipulation of Networking Firewalls
 
 The manipulation of `firewalld` to open a port for the operation of `fio` by `pbench-fio` has been removed.  The user of `pbench-fio` must have already ensured that the ports specified for each client are open.  If using the default port (as configured in a `/opt/pbench-agent/config/pbench-agent.cfg` file), then the user should ensure that port (default is `8765`) is open.  If using the `--unique-ports` option, then the user should ensure that the range of ports for the number of clients are open (e.g. for `pbench-fio --clients=hosta,hostb,hostc`, port 8765 on `hosta`, port 8766 on `hostb`, and port 8767 on `hostc`).
 
-Similarly, `pbench-uperf` no longer stops the host `firewalld` service before attempting to drive a `uperf` run.  The user should either stop the `firewalld`, or open the ports used by uperf in `firewalld` (starting port is 20,000, then incremented by 10 for every server specified).
+Similarly, `pbench-uperf` no longer stops the host `firewalld` service before attempting to drive a `uperf` run.  The user should either stop the `firewalld` or open the ports used by `uperf` in `firewalld` (starting port is 20,000, then incremented by 10 for every server specified).
 
-Finally, the Ansible Galaxy collection provided for installing the Pbench Agent provide roles for manipulating `firewalld` to enable the operation of the Tool Meister Sub-System, but those roles are not used by default.  The user must deliberately use those roles so that they no fire wall manipulation occurs without their consent.
+Finally, the Pbench Agent Ansible Galaxy collection provides roles for manipulating `firewalld` to enable the operation of the Tool Meister Sub-System, but those roles are not used by default.  The user must deliberately use those roles so that no fire wall manipulation occurs without their consent.
 
 
 ## Removal of Gratuitous Software Installation
 
-The software required to run a particular benchmark, or a particular tool, is no longer gratuitously installed during the execution of a benchmark, or during tool registration.  If a benchmark or tool requires a certain version of software to be present, those checks will be performed and reported to the user as an error if the requirements are not met.
+The software required to run a particular benchmark, or a particular tool, is no longer gratuitously installed during the execution of a benchmark, or during tool registration.  If a benchmark convenience script or tool requires a certain version of software to be present, those checks will be performed and reported to the user as an error if the requirements are not met.
```

Comment:
s/particular benchmark/particular benchmark workload/
s/execution of a benchmark/execution of the workload/